gradle.properties

```diff
@@ -12,6 +12,7 @@ org.gradle.jvmargs=-Xmx4g -Xms512M -XX:MaxPermSize=2048m -XX:MaxMetaspaceSize=1g
 org.gradle.configureondemand=true
 org.gradle.parallel=true
 org.gradle.vfs.watch=true
+org.gradle.caching=true
```

Comment:
I have heard that enabling cache can be slower on the CI because it can take more time to retrieve and unzip the cache than download everything again. Do you have any thoughts on it?

Reply:
this gradle flag enables the build cache, which relative to the existing gradle cache we're already saving on the CI is only about 300MB~ more 

Current CI `~1925 MB` https://github.com/vector-im/element-android/runs/8282616819?check_suite_focus=true

With build caching `~2197 MB` https://github.com/vector-im/element-android/runs/8309095369?check_suite_focus=true

is about 15 seconds slower~